Makefile

```diff
@@ -28,10 +28,10 @@ test_lint:
 # for more than once in Travis. Change `make test` in .travis.yml
 # to `make test_multi`.
 test_multi:
-	cd network; \
+	cd services/skipchain; \
```

Comment:
Travis Env Var would still solve the problem elegantly but for this PR then we should at least rename it something like `test_playground` or stg explicit, with an appropriate comment as you just described above.